pkg/agent/agent.go

```diff
@@ -325,8 +331,8 @@ func (c *agentController) Reconcile(ctx context.Context, req ctrl.Request) (ctrl
 	c.log.Info(fmt.Sprintf("Reconciling hostedcluster secrect %s", req))
 	defer c.log.Info(fmt.Sprintf("Done reconcile hostedcluster secrect %s", req))
 
-	// Update the AddOnPlacementScore resource
-	c.CreateAddOnPlacementScore()
+	// Update the AddOnPlacementScore resource, continue reconcile even if error occurred
+	_ = c.CreateAddOnPlacementScore(ctx)
```

Comment:
Should we be logging that an error occurred for debugging, even though we continue??

Reply:
Errors are logged in the CreateAddOnPlacementScore function.